apluslms_roman/builder.py

```diff
@@ -60,11 +65,13 @@ def __init__(self, backend_class=None, settings=None):
 
         name = getattr(backend_class, 'name', None) or backend_class.__name__.lower()
         env_prefix = name.upper() + '_'
-        env = {k: v for k, v in environ.items() if k.startswith(env_prefix)}
+        env = load_from_env(env_prefix, '.')
+        logger.debug("env without reading global config:{}".format(env))
         if settings:
             for k, v in settings.get(name, {}).items():
                 if v is not None and v != '':
-                    env[env_prefix + k.replace('-', '_').upper()] = v
+                    env[k] = v
+        logger.debug("env after read from global config:{}".format(env))
```

Comment:
Always use logging with syntax: `logger.debug("old style %s format string", env)`. This way the formatting is not done unless the log line is actually printed somewhere. Code above does always call string.format, also when debug logs are disabled.